packages/core/src/components/QueryBuilder/queryToMongo.tsx

```diff
@@ -16,7 +16,6 @@ function isAnd(rule) {
 
 const groupHasNoSubGroupsOrJustOneOR = (group) => {
   let foundOr = false;
-  // eslint-disable-next-line no-plusplus
```

Comment:
Could likely be refactored into a simpler & easier to read iteration, instead of a loop. Worst case scenario, a `forEach`